edx_rest_api_client/auth.py

```diff
@@ -17,18 +17,19 @@ def __init__(self, username, full_name, email, signing_key, issuer=None, expires
         self.tracking_context = tracking_context
 
     def __call__(self, r):
-
+        now = datetime.datetime.utcnow()
         data = {
             'username': self.username,
             'full_name': self.full_name,
             'email': self.email,
+            'iat': now,
```

Comment:
If the `exp` claim is missing, the token never expires, which may be desired in some cases. If the value were set to now, the token would be expired by the time it was processed by the recipient.